.azure-pipelines/steps/install-dotnet-6-sdk.yml

```diff
@@ -5,13 +5,13 @@ parameters:
 
 steps:
 - task: UseDotNet@2
-  displayName: install dotnet core sdk 5
+  displayName: install dotnet core sdk 6
```

Comment:
Maybe we should rename some of the variables + file names here? 

* `dotnetCoreSdk6Version` -> `dotnetCoreSdkLatestVersion`
* `install-dotnet-6-sdk.yml` -> `install-dotnet-sdk.yml` or `install-latest-dotnet-sdk.yml`?
* `install-dotnet-sdks.yml` -> `install-all-dotnet-sdks.yml` (if we use `install-dotnet-sdk.yml` above)

Reply:
I renamed dotnetCoreSdk6Version and install-dotnet-6-sdk.yml in https://github.com/DataDog/dd-trace-dotnet/pull/1885/commits/242c3c813e05991fdf1543aa6f7574ce577c6805